### ISSUE TITLE
CustomCheck messages should respect NewLines

### ISSUE THREAD

**Issue by danielmarbach** [Particular/ServicePulse#658]
While implementing a meaningful custom check failed message @boblangley and I realized that ServicePulse doesn't respect NewLines added to the custom check failed messages. It would be beneficial and helpful if the UI would respect newlines and properly render it in a responsive way

![image](https://user-images.githubusercontent.com/174258/46102588-fc975500-c1ce-11e8-90d6-e5d8fd5a076c.png)

Actual message

```
Unable to read the Dead Letter Queue length. The performance counter with category 'LocalizedCategoryName' and name 'LocalizedCounterName' and instance name 'LocalizedCounterInstanceName' is not available. It is possible that the counter category, name and instance name have been localized into different languages. Consider overriding the counter category, name and instance name in the application configuration file by adding:
   <appSettings>
     <add key="ServiceControl/Msmq/PerformanceCounterCategoryName" value="LocalizedCategoryName" />
     <add key="ServiceControl/Msmq/PerformanceCounterName" value="LocalizedCounterName" />
     <add key="ServiceControl/Msmq/PerformanceCounterInstanceName" value="LocalizedCounterInstanceName" />
   </appSettings>
```

**Comment by WilliamBZA** [comment]
@Particular/servicepulse-maintainers I've added this for our next maintenance release

**Comment by sergioc** [comment]
Fixed in upcoming v1.15 release
